package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fastify/etag",
-  "version": "4.2.0",
+  "version": "5.0.0",
```

Comment:
please revert this, I'll bump the major when releasing.

Reply:
oh, didn't know that, will do. If we retain fnv1a it's also not breaking anyway (I guess a minor bump would be fine).